installer/dev/main.cpp

```diff
@@ -103,6 +108,11 @@ int wmain(int argc, wchar_t *argv[])
 
             std::wcerr << "One or more install operations failed. Result: 0x" << std::hex << deployPackagesResult << std::endl;
         }
+
+        if (!Security::IntegrityLevel::IsElevated())
+        {
+            std::wcout << std::endl <<  "WARNING: Since WindowsAppRuntimeInstaller.exe is not run elevated, WindowsAppSDK packages are not provisioned for all users on your machine." << std::endl;
```

Comment:
WARNING is a bit strong

What's the normal text when provisioning packages? If "Provisioing: OK" or "Provisioning: ERROR 0x12345678 blahblah"  then you could make this "Provisioning: Skipped (requires elevation)".

Or simply say nothing. Only mention Provisioning if _DOING_ elevation

Reply:
Agree. Changed it no to say "Provisioning Skipped as it requires elevation"